src/clientrpc.cpp

```diff
@@ -0,0 +1,181 @@
+/******************************************************************************\
+ * Copyright (c) 2004-2020
+ *
+ * Author(s):
+ *  Volker Fischer
```

Comment:
Unless you've copied code from Volker, you don't need his name in new code.  `#include "client.h"` in the `clientrpc.h` doesn't count as copying his code.

Reply:
Ok, will update the comments. Being unfamiliar with C++ I just copied an existing file and ended up deleting most of the code :joy: